docs/tutorials/NMR/Examples_3_-Alanine_and_Silicates.md

```diff
@@ -0,0 +1,222 @@
+# Examples 3 - Alanine and Silicates
+
+We now look at some more realistic and complex structures - the amino acid alanine and 2 different silicates: cristoballite and qartz. These are fairly large crystals - to get them
+to complete in a short time we will run them on a cluster.
+**Use the following number of
+CPUs:**
+
+* analine - 4
+* cristoballite - 6
+* quartz - 13
+
+(these numbers are chosen to give efficient k-point scaling)
+
+We will also perform EFG calculations for the above structures and examine the respective data.
+
+<p style="border-width:3px; border-style:solid;"><i>
+<b>Oxygen-17 NMR</b>
+
+
+Oxygen is a component of many geological materials. Oxygen is
+also important element in organic and biological molecules since it is often intimately involved in hydrogen bonding. Solid State <sup><small>17</small></sup>O NMR should be a uniquely valuable probe as the chemical shift range of <sup><small>17</small></sup>O covers almost 1000 ppm in organic molecules. Furthermore <sup><small>17</small></sup>O has spin I = 5/2 and hence a net quadrupole moment. As a consequence of this the solid state NMR spectrum is strongly affected by the electric
+field gradient at the nucleus.
+
+Because the isotopic abundance of <sup><small>17</small></sup>O is very low (0.037%) and the NMR linewidths due to the electric field gradient relatively large, only limited Solid State NMR data is
+available. This is particularly true for organic materials. First principles calculations of <sup><small>17</small></sup>O NMR parameters have played a vital role in assigning experimental spectra, and developing empirical rules between NMR  parameters and local atomic structure.</i>
+</p>
```

Comment:
Probably want Markdown not HTML formatting

```suggestion
### Oxygen-17 NMR

Oxygen is a component of many geological materials. Oxygen is
also important element in organic and biological molecules since it is often intimately involved in hydrogen bonding. Solid State ^17^O NMR should be a uniquely valuable probe as the chemical shift range of ^17^O covers almost 1000 ppm in organic molecules. Furthermore ^17^O has spin I = 5/2 and hence a net quadrupole moment. As a consequence of this the solid state NMR spectrum is strongly affected by the electric
field gradient at the nucleus.

Because the isotopic abundance of ^17^O is very low (0.037%) and the NMR linewidths due to the electric field gradient relatively large, only limited Solid State NMR data is
available. This is particularly true for organic materials. First principles calculations of ^17^O NMR parameters have played a vital role in assigning experimental spectra, and developing empirical rules between NMR  parameters and local atomic structure.
```

Reply:
In the original pdf this was in a black box so I'm not sure if it needs to be for any particular reason (and I don't think you can do that in markdown) - I'll change it to that for now but can always revert